lib/bundler/cli/install.rb

```diff
@@ -190,5 +154,64 @@ def check_for_options_conflicts
         raise InvalidOption.new(error_message)
       end
     end
+
+    def check_trust_policy
+      if options["trust-policy"]
+        unless Bundler.rubygems.security_policies.keys.include?(options["trust-policy"])
+          Bundler.ui.error "Rubygems doesn't know about trust policy '#{options["trust-policy"]}'. " \
+            "The known policies are: #{Bundler.rubygems.security_policies.keys.join(", ")}."
+          exit 1
+        end
+        Bundler.settings["trust-policy"] = options["trust-policy"]
+      else
+        Bundler.settings["trust-policy"] = nil if Bundler.settings["trust-policy"]
+      end
+    end
+
+    def normalize_groups
+      Bundler.settings.with    = [] if options[:with] && options[:with].empty?
+      Bundler.settings.without = [] if options[:without] && options[:without].empty?
+
+      with = options.fetch("with", [])
+      with |= Bundler.settings.with.map(&:to_s)
+      with -= options[:without] if options[:without]
+
+      without = options.fetch("without", [])
+      without |= Bundler.settings.without.map(&:to_s)
+      without -= options[:with] if options[:with]
+
+      options[:with]    = with
+      options[:without] = without
+    end
+
+    def normalize_settings
+      Bundler.settings[:path]     = nil if options[:system]
+      Bundler.settings[:path]     = "vendor/bundle" if options[:deployment]
+      Bundler.settings[:path]     = options["path"] if options["path"]
+      Bundler.settings[:path] ||= "bundle" if options["standalone"]
```

Comment:
the alignment in this method is now off
also, it'd be nice to have a blank line between the different settings?